src/app/modules/status.rs

```diff
@@ -79,7 +79,7 @@ pub(crate) struct StatusModuleVTable {
     /// 
     /// ### Behavior (Fighter, Weapon, and Item)
     /// This method will:
-    /// 1. Enable status execution and start by transitioning the fighter to [`app::StatusKind::None`]
+    /// 1. Enable status execution and start by transitioning the fighter to [`app::FighterStatusKind::None`]
```

Comment:
I will accept this for now but in the future I plan on changing how I declare status kinds in this repo. But given I've been relatively inactive this is fine for me.

Reply:
Really I only changed it because there was no type `StatusKind` in the crate. I originally considered replacing all instances of `FighterStatusKind` with `StatusKind`, but ultimately decided against it seeing how the status kind constants all start with `FIGHTER_STATUS_KIND_`.